torch/_refs/__init__.py

```diff
@@ -4241,12 +4255,17 @@ def movedim(
     if type(destination) is int:
         destination = (destination,)
 
+    # Convert to list to produce a compatible error message with core PyTorch,
+    # which prints sequences in square brackets.
+    def _format_seq(seq):
+        return list(seq)
+
```

Comment:
Allocates a list, but it's only done in the error path. Also, probably better to do it this way instead of defining some custom formatting function because that would still allocate and might miss some corner cases(?).

Reply:
I wouldn't bother defining this helper at this time and just make a note about why there are explicit `list` calls in the string